src/components/StoreInfo/StoreLocation.tsx

```diff
@@ -17,15 +17,56 @@ const StoreLocation: FC<Props> = ({ locations }) => {
                   className={styles.wrap__singleLocation}
                   key={location.seller_id}
                 >
-                  <address>
-                    <p className={styles.address}>
-                      {location.address1 + (location.address2 || '')}
+                  <address
+                    itemProp="address"
+                    itemType="https://schema.org/PostalAddress"
+                  >
+                    <p itemProp="streetAddress" className={styles.address}>
+                      {location.address1}
+                      {location.address2 ? <br /> : ''}
+                      {location.address2 ? location.address2 : ''}
                     </p>
                     <p className={styles.address}>
-                      {location.city}, {location.state} {location.zip_code}
+                      <span itemProp="addressLocality">{location.city}</span>,{' '}
+                      <span itemProp="addressRegion">{location.state}</span>{' '}
+                      <span itemProp="postalCode">{location.zip_code}</span>
                     </p>
                   </address>
-                  <p className={styles.address}>{location.phone_number}</p>
+                  {location.address1 && (
+                    <span>
+                      <a
+                        href={
+                          'https://www.google.com/maps/place/' +
+                          (
+                            (location.address1 + ' ' || '') +
```

Comment:
not quite sure how, but I feel like there might be a better way to handle this? maybe with a `map` or `forEach` looping through the properties, and doing the concatenation on each step

Reply:
And also for the places where you add space( `' '`), that needs to be escaped/ replaced with the unicode version of space, `%20` I think.

Reply:
Google Maps seemed to be using + as the separators in the links, but also takes encoded spaces. I couldn't get a encodeURI or encodeURIComponent working though, which would be preferable, anyone know the equivalent?

Reply:
Oh that's interesting, I guess it'd be "better" to use `+` to be consistent to how google maps search works, but since maps automagically does whatever escaping it wants, it really doesn't matter.